### PR TITLE
First draft

### PR DESCRIPTION
**Pull request by john-0x**

### CODE REVIEW

File: chapters/asset_registrar.md [hunk 1]
```diff
@@ -1,4 +1,66 @@
...
+
+The Asset Registrar stores data about each investment-eligible asset on-chain.
+
+The Asset Registrar holds the following public mapping:
+
+  `mapping (address => Asset) public information;`
+
+The Asset Registrar is primarily a smart contract to hold and provide maintenance infrastructure for a list of investment-eligible assets.  Assets have the following characteristics:
+
+| Name | Type | Description |
```
Comment (jennazenk) [inline chapters/asset_registrar.md]: table is repeated twice @john-0x

File: chapters/asset_registrar.md [hunk 2]
```diff
@@ -1,4 +1,66 @@
...
+| name | bytes32 | Human-readable name of the Asset as found in the ERC223 token standard |
+| symbol | bytes8 | Human-readable symbol of the Asset as found in ERC223 token standard |
+| decimal | uint | Precision specified as the number of decimals |
+| url | string | url address for Asset's extended information |
+| ipfsHash | string  | ipfs  address for Asset's extended information |
+| chainId | bytes32 | Chain where the asset resides |
+|breakIn | address | Address  of  break-in  contract  on destination chain |
+| breakOut | address | Address  of  break-out contract on the native chain |
+
+add more detail on: AR 1st, then price feed...
```
Comment (jennazenk) [inline chapters/asset_registrar.md]: would delete internal comments/notes

File: chapters/exchange.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+#### Exchange Adapters
+
+Exchange Adapters are smart contracts which communicate directly and on-chain with the intended DEX smart contract. They serve as a translation bridge between the Melon Fund and the DEX.
+
+Currently, the Melon Protocol has adapters to integrate the following DEXs:
+
+- Oasis DEX
+- RadarRelay (0x-based)
+- ERCDex (0x-based)
```
Comment (jennazenk) [inline chapters/exchange.md]: Instead of: 
- RadarRelay (0x-based)
- ERCDex (0x-based)

I would say 

- 0x protocol (which gives access to all the 0x relayers)

File: chapters/price_feed.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+Data which is exogenous to the blockchain must be actively provided by a third party. Asset price data is critical to fund management and Melon Funds. Current and accurate price data enables real-time valuation and indispensible context for investment management decisions.  As such, careful attention must be taken to *how* this exogenous data enters the Melon Fund platform.
+
+Independent parties wishing to provide the service of an asset price feed may do so by running software which gathers asset price data (corresponding to the Melon Asseet Universe set forth in the Melon Asset Registrar) and reliably feeding this data in a specifed format to the Melon Platform.
+
+#### Operator Staking
+
+Prospective Price Feed Operators (candidates) must offer a deposit or "stake" in Melon Tokens (MLN). Individual Price Feed Operators are then "selected" by the Melon Platform solely on the level of their stake. The staked MLN is held securely in custody by a smart contract which also determines which of the candidates is actually feeding price data to the Melon Fund Platform.
```
Comment (jennazenk) [inline chapters/price_feed.md]: >"selected" by the Melon Platform 

would rephrase : "selected" by the Melon contracts

**Review by jennazenk — CHANGES_REQUESTED** [review]

**Review by jennazenk — APPROVED** [review]

**Review by travs — APPROVED** [review]

**Review by GauthamN — APPROVED** [review]
